sopel/module.py

```diff
@@ -199,6 +209,117 @@ def add_attribute(function):
     return add_attribute
 
 
+def find(*patterns):
+    """Decorate a function to be called each time patterns is found in a line.
```

Comment:
```suggestion
    """Decorate a function to be called for each time a pattern is found in a line.
```

This isn't strictly a nemesi**S**, but it is kinda awkward when phrased with the plural.